teste2e/author_submit_and_watch_extrinsic_test.go

```diff
@@ -82,7 +82,7 @@ func TestAuthor_SubmitAndWatchExtrinsic(t *testing.T) {
 
 	var accountInfo types.AccountInfo
 	ok, err = api.RPC.State.GetStorageLatest(key, &accountInfo)
-	if err != nil || !ok {
+	if err != nil {
 		panic(err)
 	}
```

Comment:
Does this `ok` go unused?

Reply:
Yes, because it would error every time an account was used for the first time. I have no clue how their tests passed though. Maybe their runtime gives a default value? Another possibility is that they rely on the fact that go tests are sequential so they know for a fact that the test running in CI would be executed in order and thus Alice would have been guaranteed to have made a transaction before this.

Reply:
I mean we are not changing the way that  we are testing from go-substrate-rpc-client right? I'm fine to let this slide if you're confident that you don't need the `ok` check. I have come across the same issue with checking the balances of new accounts in that `ok` would be false. I have shorted the logic to defaulting this to being zero balance but didn't investigate very far. 

You can add a unit (`_`) instead of ok if you like.

Reply:
> I mean we are not changing the way that we are testing from go-substrate-rpc-client right?

> You can add a unit (_) instead of ok if you like.

Sounds like a good fix for now.

Reply:
I use slack as transient communication and github as an archive tool? Just adding from slack that this was an 'investigate later' sort of issue on the way substrate vs cennznet deem what is `ok`